.storybook/manager-head.html

```diff
@@ -20,7 +20,7 @@
 <script>
   setTimeout(() => {
     if (window.CONFIG_TYPE === 'DEVELOPMENT') {
-      document.body.classList.add('show-interaction-tests');
+      document.body.classList.add('show-interaction-test');
```

Comment:
This can be the same as before

Reply:
And change this in line 7:
```
body:not(.show-interaction-tests) [data-parent-id^='interaction-test'] {
```

The `^` means "startWith". So then we are sure that everything will be hidden when it starts with interaction-test